tooling/noir_js/src/input_validation.ts

```diff
@@ -0,0 +1,55 @@
+// Check if all of the input values are correct according to the ABI
+export function validateInputs(inputs: any, abi: any) {
```

Comment:
This method was added because when we supply a string and the abi expects a number, the error is not that useful. This input validation code can live in noirc_abi instead of here.

For the future, we can add code that takes in an ABI and code generates a typescript struct that has proper type safety. Since the abi requires things like u64, we can additionally codegen a .validate method on the struct which will check if the bigint fits within a u64 for example.